src/peft/utils/loftq_utils.py

```diff
@@ -0,0 +1,190 @@
+import torch
```

Comment:
AFAICT, this code is mostly the same as from your LoftQ repo. Could you please add a reference?